crates/bevy_reflect/bevy_reflect_derive/src/derive_data.rs

```diff
@@ -2,12 +2,13 @@ use crate::container_attributes::ReflectTraits;
 use crate::field_attributes::{parse_field_attrs, ReflectFieldAttr};
 use crate::utility::members_to_serialization_denylist;
 use bit_set::BitSet;
+use proc_macro2::TokenStream;
```

Comment:
We can; but in general I think the common practice is to `improt proc_macro2::TokenStream` everywhere (since it's compatible with `syn` and `quote`) and only at the very last stage convert into a `proc_macro::TokenStream`.

I think most crates have this model; where `proc_macro::TokenStream` is the only one that is explicitly spelt out

Reply:
Basically, I think that `use proc_macro2::TokenStream` should be used everywhere by default (since it's the one we apply ops on), and then we just call `proc_macro::TokenStream` when needed. But this can be left to another PR, ot unblock this one

Reply:
Yeah I agree, it makes more sense to flip it. But just for consistency until a cleanup PR comes along to handle it, we should try to keep it consistent